src/app/views/user/user.html

```diff
@@ -85,6 +85,9 @@ <h3 class="panel-title">{{'mnoe_admin_panel.dashboard.user.geo_information_title
       <div class="col-sm-12 top-buffer-2">
         <mnoe-organizations-local-list list="vm.user.organizations" user="vm.user" view="all"></mnoe-organizations-local-list>
       </div>
+      <div class="col-sm-12 top-buffer-2">
+        <mnoe-dashboards-list ng-if="vm.isAdminDashboardViewingEnabled"></mnoe-organizations-local-list>
```

Comment:
looks like a little copy and paste :trollface: - opening and closing tags don't match

Reply:
I would never copy and paste code.